### ISSUE TITLE
Update README

### ISSUE THREAD

**Issue by Mqxx** [chrisant996/clink-fzf#8]
Your README is not up to date with the lua script...

https://github.com/chrisant996/clink-fzf/blob/main/fzf.lua#L24-L29

**Comment by chrisant996** [comment]
@Mqxx Please be specific about which part of the README, and in what way.

It looks accurate to me, and I don't want to have to spend a long time figuring out what detail you noticed -- can you please share it?  🙂

**Comment by Katze719** [comment]
hey, `"\e[27;5;32~": "luafunc:fzf_complete_force" # Ctrl+Space uses fzf to filter match completions (and supports '**' for recursive).` is not mentioned in the readme, that confused me

**Comment by chrisant996** [comment]
Ok thanks.

**Comment by Mqxx** [comment]
This was actualy not the problem... Like I mentioned in my original post these lines are diffrent.

[In the lua file](https://github.com/chrisant996/clink-fzf/blob/main/fzf.lua#L24-L29):
```
# Default key bindings for fzf with Clink.
"\C-t":        "luafunc:fzf_file"           # Ctrl+T lists files recursively; choose one or multiple to insert them.
"\C-r":        "luafunc:fzf_history"        # Ctrl+R lists history entries; choose one to insert it.
"\M-c":        "luafunc:fzf_directory"      # Alt+C lists subdirectories; choose one to 'cd /d' to it.
"\M-b":        "luafunc:fzf_bindings"       # Alt+B lists key bindings; choose one to invoke it.
"\t":          "luafunc:fzf_complete"       # Tab uses fzf to filter match completions, but only when preceded by '**' (recursive).
"\e[27;5;32~": "luafunc:fzf_complete_force" # Ctrl+Space uses fzf to filter match completions (and supports '**' for recursive).
```

[In the README file](https://github.com/chrisant996/clink-fzf/blob/main/README.md?plain=1#L52-L58):
```
# Default key bindings for fzf with Clink.
"\C-t":        "luafunc:fzf_file"       # Ctrl+T lists files recursively; choose one or multiple to insert them.
"\C-r":        "luafunc:fzf_history"    # Ctrl+R lists history entries; choose one to insert it.
"\M-c":        "luafunc:fzf_directory"  # Alt+C lists subdirectories; choose one to 'cd /d' to it.
"\M-b":        "luafunc:fzf_bindings"   # Alt+B lists key bindings; choose one to invoke it.
"\t":          "luafunc:fzf_complete"   # Tab uses fzf to filter match completions.
"\e[27;5;32~": "luafunc:fzf_complete"   # Ctrl+Space uses fzf to filter match completions.
```

The function name (`luafunc:fzf_complete -> luafunc:fzf_complete_force`) is diffrent.

**Comment by Mqxx** [comment]
> @Mqxx Please be specific about which part of the README, and in what way.
> 
> It looks accurate to me, and I don't want to have to spend a long time figuring out what detail you noticed -- can you please share it? 🙂

@chrisant996 In my original post I have marked the lines that are diffrent.

**Comment by chrisant996** [comment]
> @chrisant996 In my original post I have marked the lines that are diffrent.

Yes, that caused the confusion:

6 lines were listed from the .lua file, not the README.md file.  It didn't mention which lines in the README.md file were expected to be updated.  So I had to make a guess.  I looked for the first lines in the README.md file that referred to the key bindings.

The first 4 lines out of the 6 are correct to be different, when comparing to the first text in the README.md about key bindings, because of differences between .md and .lua file syntax.  After confirming that the first 4 lines were actually correct, I stopped spending time trying to figure out what to compare against.  But now it's clear you had been looking at other lines in the README.md file.

After the follow-up reply, once I noticed one difference, I figured probably only one difference was being reported.  But it turns out there were 5 specific subtle differences -- 2 in the first place in the README.md that talks about key bindings, and 3 in the second place.

It helps very much when reports describe what went wrong.  For example, "I was confused because the wrong command name is listed".  That would let me know exactly what the problem is and save a lot of time and confusion, but "6 lines are different" doesn't end up being very clear, especially when it cites lines from a different file than the file with the problem.
